socorro/unittest/external/postgresql/test_crashstorage.py

```diff
@@ -297,13 +294,13 @@ def test_basic_postgres_save_processed_success_3_truncations(self):
         crashstorage.save_processed(a_processed_crash_with_everything_too_long)
 
         assert mocked_database_connection_source.call_count == 1
-        assert mocked_cursor.execute.call_count == 3
+        assert mocked_cursor.execute.call_count == 2
```

Comment:
Calling `save_processed` used to save to reports tables, then plugins tables, then the processed crashes tables. Now it just does the first two. Thus this goes from 3 to 2.